compose.yaml

```
@@ -0,0 +1,43 @@
+services:
+  frontend:
+    image: frontend:dev
+    build:
+      context: frontend
+      dockerfile: Dockerfile
```

Importantly, I think we're missing `target` arguments here, so if you haven't already built the image (which may be why you aren't seeing this!) then when it builds it just uses the final stage in the Dockerfile, which you get away with for the frontend but means the `backend` service actually runs your tests!
```suggestion
      dockerfile: Dockerfile
      target: development
```

---

Ah gotcha, yeah I think I'd seen an issue with that and then it stopped happening. That explains it.